### PR TITLE
MIMIR 797 local search filter

### PR DESCRIPTION
**Pull request by johnnadeluy**

### CODE REVIEW

File: src/main/resources/site/parts/localSearch/localSearch.es6 [hunk 1]
```diff
@@ -25,14 +20,16 @@ exports.get = function(req) {
...
+function renderPart(req) {
   const part = getComponent()
-  const items = getChildren({
+  const filteredItems = getChildren({
     key: part.config.searchFolder,
     start: 0,
     count: 1000,
     sort: 'displayName ASC'
-  }).hits.map((item) => {
+  }).hits.filter((content) => content.type === `${app.name}:statistics` || content.type === `${app.name}:article` || content.type === `${app.name}:page`)
```
Comment (Garlov) [inline src/main/resources/site/parts/localSearch/localSearch.es6:30]: It's better and faster to do the filter in the search config, not on the result

**Review by Garlov — CHANGES_REQUESTED** [review]
From the description in jira, should it be possible for the content-studio user to select the filter manually, or is it supposed to be always on?

And change from result filter to config filter on content-type

**Comment by johnnadeluy** [comment]
I believe that the filter is supposed to always be on, so that folders and other content don't get displayed in the results.

**Review by Garlov — APPROVED** [review]
Nice 👍

**Pull request merged**